### DISCUSSION TITLE
GTK4 LibAdwaita Support

### DISCUSSION THREAD

**Discussion by tdtooke** [linuxdeepin/developer-center#5086] [Features Request & Ideas | 特性请求 & 头脑风暴]
Hi.  I use Plasma with a Deepin inspired theme.  I use this for GTK3 apps.  If this is something you have time to do I think it'd be great to have a libadwaita compatible gtk4 theme also.  I say libadwaita because now most apps which use gtk4 are libadwaita.

**Comment by BLumia** [comment]
Hi,

> I use Plasma with a Deepin inspired theme

If you means plasma theme, did you made a deepin-inspired theme? If not, which theme are you using, could you provide a link to it?

> I think it'd be great to have a libadwaita compatible gtk4 theme

I agree, although we might have time to do such thing after the deepin v23 stable release, so contributions are also welcome :)

**Reply by tdtooke** [reply]
I think it's this one: https://store.kde.org/p/1410706 .  There are a few others on the kde store.  I use the discover app to update my themes and I'm at work so I'm not sure if this is the one because there are others.  I can't contribute as of yet, I'm very weak on gtk stuff like a lot of plasma users.

**Reply by BLumia** [reply]
I understand. and btw there is a https://stopthemingmy.app/ site you might want to check out. libadwaita-based applications are technically theme-able but preferred not to be themed by user, so we are actually not sure if that's something we need to do.
